ozark/pom.xml

```diff
@@ -79,13 +120,37 @@
                     </execution>
                 </executions>
             </plugin>
+            <!--
+             This plugin is reponsible for packaging artifacts
+             as OSGi bundles.  Please refer to
+             http://felix.apache.org/site/apache-felix-maven-bundle-plugin-bnd.html
+             for more information about how to use this plugin.
+           -->
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
```

Comment:
I agree with @lefloh. The version isn't set in the parent, so Maven will always try to use the latest released one. But this makes the build unpredictable. That's what the Maven warning basically is telling us.

Jersey for example sets the version of the plugin in it's own parent pom:

https://github.com/jersey/jersey/blob/master/pom.xml#L558-578

I really think omitting Maven plugin versions is bad practice (unless you inherit them from a parent). So I think we should definitely set the version.